pymc3/sampling.py

```diff
@@ -1294,15 +1294,17 @@ def sample_prior_predictive(samples=500, model=None, vars=None, random_seed=None
         Number of samples from the prior predictive to generate. Defaults to 500.
     model : Model (optional if in `with` context)
     vars : iterable
```

Comment:
shouldn't we call this `varnames` like elsewhere?

Reply:
If that's a general convention, I could put a new PR for this.
Also, a follow-up question that we could use to clarify the docstring - am I correct in thinking that `sample_prior_predictive()` will ignore any observations that are in the model?  If so, we could extend the docstring to make this clear.